splash/lua_modules/splash.lua

```diff
@@ -132,5 +132,21 @@ function Splash:jsfunc(...)
   return unwraps_errors(func)
 end
 
+function Splash:__newindex( index, value )
+  if index == "js_enabled" then
+    self.members[index] = value
+    self:set_js_enabled(value)
+  else
```

Comment:
It'd be nice to add `splash.images_enabled` as well, for consistency, and maybe also `splash.cookies`, `splash.result_content_type`, `splash.navigation_locked`, etc. But it is not easy to add them right now - names of these attributes have to be listed in several places, etc. I'm not asking to add them all in this PR, but the code should allow that. There should be an obvious way to add an attribute (like we can add a new method by creating a Python method and marking it as `@command`).